stable_baselines3/common/base_class.py

```diff
@@ -542,21 +542,24 @@ def learn(
     def predict(
         self,
         observation: np.ndarray,
-        state: Optional[np.ndarray] = None,
-        mask: Optional[np.ndarray] = None,
+        state: Optional[Tuple[np.ndarray, ...]] = None,
+        episode_start: Optional[np.ndarray] = None,
         deterministic: bool = False,
-    ) -> Tuple[np.ndarray, Optional[np.ndarray]]:
+    ) -> Tuple[np.ndarray, Optional[Tuple[np.ndarray, ...]]]:
         """
-        Get the model's action(s) from an observation
+        Get the policy action and state from an observation (and optional hidden state).
```

Comment:
Still bit confused what is the "and state from an observation"? It only returns policy action and an optional hidden state?